web/cypress/integration/roadmap-mobile.spec.ts

```diff
@@ -30,38 +34,47 @@ describe("Roadmap [feature] [all]", () => {
     cy.get('[data-task="research-insurance-needs"]').should("exist");
 
     // step 3
+    cy.get('[id="start-content"]').should("be.visible");
+    cy.get('[id="start-header"]').click({ force: true });
+    cy.get('[id="start-content"]').should("not.be.visible");
+    cy.get('[id="start-header"]').click({ force: true });
+    cy.get('[id="start-content"]').should("be.visible");
     cy.get('[data-step="3"]').should("exist");
     cy.get('[data-task="register-trade-name"]').should("exist");
 
     // step 4
     cy.get('[data-step="4"]').should("exist");
 
+    // step 5
+    cy.get('[data-step="5"]').should("exist");
+
     // tasks screen
-    cy.get('[data-task="register-trade-name"]').click();
+    cy.get('[data-task="register-trade-name"]').click({ force: true });
```

Comment:
why do we need all the force-true statements now?

Reply:
Cypress seemed to be skipping some click events.

Reply:
@aloverso We are experiencing some weird behavior with Cypress. It will click a button, but the screen doesn't change and leads to a failed test. We did some research and think the option passed into the click method would address the instances were cypress is failing. 

I've also added a new Cypess command - forceClick, which would apply the click with the options.

Reply:
Huh, strange, but yeah if this helps then that's fine